Header header
bool engaged
float32 speed
float32 steer
float32 torque
bool rate_mode
bool crawling_mode
